### PR TITLE
Miozzijd_codeReview2

### PR DESCRIPTION
**Pull request by Jamiozzi**
**Analysis of the program:** 
Group Scheduler is an app for event management/planning. Users can create events and add them to schedule as a form of event management.

**Was the program available in UC Github on time?**
Yes the program was available on time.

**Is the program documented/commented well enough for you to understand?**
I found the project easy enough to understand.

**Does the program compile?**
The program does compile

**Rational behind your changes:**
I changed variables to private and ordered those variables according to kotlin conventions. I renamed constant variables with uppercase letters according to kotlin conventions. I renamed tests using backticks and spaces according to kotlin conventions.

**What I learned from this code review:**
- I learned how to use swipe gestures from the implementation in this app.
- I learned that private variables are conventionally after public variables during my research for this code review.
- I leaned that constant variables are always supposed to be named in capital letters during my research for this code review.

### CODE REVIEW

File: app/src/main/java/uc/edu/klopfsea/groupscheduler/model/Event.kt [hunk 1]
```diff
@@ -12,8 +12,8 @@ data class Event(
 ) {
     // property declared in class body
     var id = 0
-    var month = 0
-    var year = 0
+    private var month = 0
+    private var year = 0
```
Comment (Jamiozzi) [inline app/src/main/java/uc/edu/klopfsea/groupscheduler/model/Event.kt:16]: these variables could be set to private, and therefore should be after any public variables according to these kotlin conventions, https://kotlinlang.org/docs/coding-conventions.html#modifiers-order

File: app/src/test/java/uc/edu/klopfsea/groupscheduler/PlannedEventUnitTest.kt [hunk 2]
```diff
@@ -6,48 +6,48 @@ import uc.edu.klopfsea.groupscheduler.Events.PlannedEvent
...
 class PlannedEventUnitTest {
 
-    var name: String = "Dinner"
-    var date: String = "2021-04-26"
-    var startTime: String = "1700"
-    var plannedEvent: PlannedEvent = PlannedEvent(name, date, startTime)
-    lateinit var invalidPlannedEven: PlannedEvent
+    private var NAME: String = "Dinner"
+    private var DATE: String = "2021-04-26"
+    private var START_TIME: String = "1700"
```
Comment (Jamiozzi) [inline app/src/test/java/uc/edu/klopfsea/groupscheduler/PlannedEventUnitTest.kt:11]: These constant strings should be named in all uppercase according to these kotlin conventions, https://kotlinlang.org/docs/coding-conventions.html#property-names

File: app/src/test/java/uc/edu/klopfsea/groupscheduler/PlannedEventUnitTest.kt [hunk 3]
```diff
@@ -6,48 +6,48 @@ import uc.edu.klopfsea.groupscheduler.Events.PlannedEvent
...
+    private var plannedEvent: PlannedEvent = PlannedEvent(NAME, DATE, START_TIME)
+    private lateinit var invalidPlannedEven: PlannedEvent
 
     @Test
-    fun plannedEvent_maintainsState() {
+    fun `planned event maintains state`() {
```
Comment (Jamiozzi) [inline app/src/test/java/uc/edu/klopfsea/groupscheduler/PlannedEventUnitTest.kt:16]: this test and the tests on lines 24, 35, and 46 names were refactored, using backticks and spaces as apposed to double qoutes and uderscores to fit the kotlin conventions described here, https://kotlinlang.org/docs/coding-conventions.html#names-for-test-methods

Comment (discospiff) [reply app/src/test/java/uc/edu/klopfsea/groupscheduler/PlannedEventUnitTest.kt:16]: Good 

(but opposed, not apposed.  :) )

**Comment by discospiff** [comment]
Good suggestions.  The group should review and consider merging.

**Pull request closed without merging**